.github/workflows/check-prs.yml

```
@@ -0,0 +1,64 @@
+name: Check Pull Requests
+
+on:
```

YAMLlint (1.35.1)

[warning] truthy value should be one of [false, true]

(truthy)